docs/Using Fleet/enroll-hosts.md

```diff
@@ -314,9 +314,27 @@ expiration setting. To configure this setting, in the Fleet UI, head to **Settin
 
 > The fleetd Chrome browser extension is supported on ChromeOS operating systems that are managed using [Google Admin](https://admin.google.com). It is not intended for non-ChromeOS hosts with the Chrome browser installed.
 
+### Overview
+Google Admin uses *Organizational Units (OUs)* to organize *devices* and *users*.
```

Comment:
Would remove these italics. Also, looks like Google doesn't capitalize "organizational units" so we don't need to here.
```suggestion
Google Admin uses organizational units (OUs) to organize devices and users.
```